### PR TITLE
Add Troubleshooting chapter

### PR DESCRIPTION
**Pull request by lategoodbye**
You are free to make suggestions for new questions ...

### CODE REVIEW

File: docs/source/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+Yes it is, but be aware that the configuration and database files will not be converted.
+
+
+How can I access the EVerest admin panel on Charge Control C?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Charge Control C doesn’t have an EVerest admin panel because of its limited resources. Please
+use your development environment to setup your configuration file or just use a plain text editor.
+
```
Comment (martinlukas84) [inline docs/source/troubleshooting.rst:44]: We can also refer to the Github repository: https://github.com/EVerest/everest-admin-panel

File: docs/source/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No, the Control Pilot interface on Charge Control C is not able to operate as an EV. Please look at
+our `website <https://www.chargebyte.com/>`_ for more suitable products.
+
+
+Is it possible to use the Charge Control C as DC charge controller?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes, for prototypes it’s possible to use the Charge Control C as DC SECC for DIN 70121 or ISO 15118.
```
Comment (t-mon) [inline docs/source/troubleshooting.rst:23]: Why as prototype but not as product? Is missing hardware the issue?

**Review by t-mon — APPROVED** [review]

File: docs/source/troubleshooting.rst (hunk 2 above)
Comment (lategoodbye) [reply docs/source/troubleshooting.rst:23]: My explanation so far, the Charge Control C was designed only for the AC use case, which results in the following disadvantages:
- unnecessary components like plug lock interface, AC contactor or second PLC chip
- lack of CAN interface or safety controller

File: docs/source/troubleshooting.rst [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+   
+How can I use CAN with Charge Control C?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no Charge Control C with onboard CAN interface available, so we suggest to use a PeakCAN
+USB adapter.
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:30]: _Note that the EVerest charging stack does not have capabilities for control via CAN yet._

(I'm not aware that it's an FAQ yet, by I guess some CAN users have that goal.

Perhaps even add:
_If you require CAN control of the charger, you need to implement your own module._

File: docs/source/troubleshooting.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes it is, but be aware that the configuration and database files will not be converted.
+
+
+How can I access the EVerest admin panel on Charge Control C?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Charge Control C doesn’t have an EVerest admin panel because of its limited resources. Please
+use your development environment to setup your configuration file or just use a plain text editor.
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:43]: setup -> set up
```suggestion
use your development environment to set up your configuration file or just use a plain text editor.
```

File: docs/source/troubleshooting.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+
+Does EVerest on Charge Control C support ISO 15118-20 yet?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ISO15118-20 implementation is currently still under development and currently not included in
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:49]: 2x _currently_. I suggest:
```suggestion
The ISO15118-20 implementation is currently still under development and not yet included in
```

File: docs/source/troubleshooting.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+our EVerest releases. The implementation is located on a 
+`test branch of the everest-core repository <https://github.com/EVerest/everest-core/tree/testing/iso15118-20>`_.
+As soon as the implementation has been merged to the main branch, we will include this module in the
+next release cycle.
+
+
+How do I set up OCPP 2.0.1 on Charge Control C with EVerest?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Unfortunately, there are some manual steps on a development PC necessary. At first checkout the
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:59]: Word order and other small details:
```suggestion
Unfortunately, some manual steps on a development PC are necessary. First, check out the
```

File: docs/source/troubleshooting.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+   git clone https://github.com/EVerest/libocpp.git
+   cd libocpp/config/v201
+
+Now adapt OCPP 2.0.1 config.json to your needs (e.g. NetworkConnectionProfiles):
+
+.. code-block:: console
+
+   gedit config.json
+
+After that you can create device model database and insert the configuration:
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:73]: _a_ or _the_ device model database

File: docs/source/troubleshooting.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+.. code-block:: console
+   
+   python3 init_device_model_db.py --db device_model_storage.db --schemas component_schemas/ init
+   python3 init_device_model_db.py --db device_model_storage.db --schemas component_schemas/ --config config.json insert
+   
+Then copy the device model database on Charge Control C (adapt IP address to your environment):
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:80]: on -> onto the
```suggestion
Then copy the device model database onto the Charge Control C (adapt IP address to your environment):
```

File: docs/source/troubleshooting.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+.. code-block:: console
+
+   scp device_model_storage.db root@<ip-address>:/var/lib/everest/ocpp201
+   
+Finally make sure the DeviceModelDatabasePath in your EVerest configuration points to
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:86]: This means the other, global configuration, so feel free to mention /etc/everest/config.yaml.

File: docs/source/troubleshooting.rst [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+/var/lib/everest/ocpp201/device_model_storage.db and then restart EVerest on the Charge Control C.
+
+
+I would like to implement a custom Modbus device in EVerest. Where should I start?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+EVerest already has module which takes care of Modbus communication. Please have a look at
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:93]: -> _a_ module
```suggestion
EVerest already has a module which takes care of Modbus communication. Please have a look at
```

File: docs/source/troubleshooting.rst [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+`SerialCommHub <https://everest.github.io/nightly/_generated/modules/SerialCommHub.html>`_.
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:94]:
```suggestion
`SerialCommHub <https://everest.github.io/nightly/_generated/modules/SerialCommHub.html>`_, and let your module interact with this module via the `serial_communication_hub` interface.
```

File: docs/source/troubleshooting.rst [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+
+Contact
+-------
+
+Support
+^^^^^^^
+
+EVerest is an open-source project with a lot of modules, which is surrounded by a big community.
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:103]: "surrounded" sort of works, but how about "supported"?

**Review by barsnick — CHANGES_REQUESTED** [review]

File: docs/source/troubleshooting.rst (hunk 10 above)
Comment (FaHaGit) [inline docs/source/troubleshooting.rst:93]: "has a  module"

File: docs/source/troubleshooting.rst [hunk 13]
```diff
@@ -0,0 +1,126 @@
...
+chargebyte is an active part of this community. However chargebyte is not able to provide support
+for every aspect of EVerest. In order to get quick answers here are some suggestions:
```
Comment (FaHaGit) [inline docs/source/troubleshooting.rst:105]: not sure, but I would add a comma "answers, here are some"

**Review by FaHaGit — CHANGES_REQUESTED** [review]

File: docs/source/troubleshooting.rst [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+
+   
+How can I use CAN with Charge Control C?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There is no Charge Control C with onboard CAN interface available, so we suggest to use a PeakCAN
+USB adapter.
+
+
+I want to control EVerest via CAN, how can i achieve this?
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:34]: Capital I:
```suggestion
I want to control EVerest via CAN, how can I achieve this?
```

File: docs/source/troubleshooting.rst [hunk 15]
```diff
@@ -0,0 +1,138 @@
...
+
+No, the Control Pilot interface on Charge Control C is not able to operate as an EV. Please look at
+our `website <https://www.chargebyte.com/>`_ for more suitable products.
+
+
+Is it possible to use the Charge Control C as a DC charge controller?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Yes, for prototypes it’s possible to use the Charge Control C as DC SECC for DIN 70121 or ISO 15118.
+But the Charge Control C was designed for AC use case in mind.
```
Comment (barsnick) [inline docs/source/troubleshooting.rst:24]: I would say "with [something] in mind:
```suggestion
But the Charge Control C was designed with the AC use case in mind.
```

**Review by barsnick — CHANGES_REQUESTED** [review]
Minor nits, otherwise fine.

**Pull request merged**